components/barcode/overview.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Overview
+page_title: Barcode Overview
+description: Overview of the Barcode for Blazor.
+slug: components/barcode/overview
+tags: telerik,blazor,barcode,overview
+published: True
+position: 0
+---
+
+# Barcode Overview
+
+The Barcode represents data in a machine-readable format. You can choose from several supported standards.
+
+All graphics are rendered on the client by using [Scalable Vector Graphics (SVG)](https://www.w3.org/Graphics/SVG/).
+
+## Basics
+
+To add a Telerik Barcode to your Blazor app:
+
+1. Add the `TelerikBarcode` tag.
+1. Inside it, add `BarcodeText` tag that styles the barcode text.
+1. Provide `Value` (one-way data binding) or `bind-Value` (two-way data binding) property
```

Comment:
match the example. Also, what is two-way binding for a non-editor component? If you want to talk about this here (I recommend mantioning in a later Features section), explain what that is in a dedicated paragraph or article.

Reply:
I agree with you here that this is not the place for mentioning two-way binding.